commands/exportcommands.py

```
@@ -338,61 +397,107 @@ def _generate_install_script(self):
+        # Check if we're in the cloned repo directory
+        if not os.path.exists('README.md'):
+            # First export
+            self.current_version = "v1.0.0"
+            self.change_summary = "Initial command system export"
+            print(f"   First export detected - Version: {self.current_version}")
+            return
+            
+        try:
+            with open('README.md', 'r', encoding='utf-8') as f:
+                current_readme = f.read()
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)